### DISCUSSION TITLE
Setup an email alert notification for Security Onion 2.80

### DISCUSSION THREAD

**Discussion by ITgeevoSOC** [Security-Onion-Solutions/securityonion#6104] [Unsupported Versions] [answered]
Hi,

We need to setup an alert notification for Security Onion 2.80 (to include elastic alerts and wazuh alerts).

We have found some old github discusions that use squil.email, but this not included in Security Onion 2.80 https://github.com/security-onion-solutions/security-onion/issues/144.

Please can you advise how to proceed with the above setup?

Thanks in advance.

**Comment by csanate** [comment, marked as answer]
Check out: https://docs.securityonion.net/en/2.3/elastalert.html

**Reply by extremepaperclip** [reply]
Question:  For the section "Email - Internal" in that doc (https://docs.securityonion.net/en/2.3/elastalert.html) What path and what file would I write this config to? 

alert:
- "email"
email:
- "youremail@yourcompany.com"
smtp_host: "your_company_smtp_server"
smtp_port: 25
from_addr: "elastalert@yourcompany.com"

**Reply by extremepaperclip** [reply]
To add some info to the above:  Is there a way to add a global config file and specify the email address for so-elastalert and/or Playbooks to send specific types of alerts/notifications to?  (Ultimately it would be great to do this kind of config within the web gui)

**Reply by dougburks** [reply]
@extremepaperclip This question has already been marked as answered. From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.
